### ISSUE TITLE
Support close match resolution in MediaPortal GUI

### ISSUE THREAD

**Issue by GoogleCodeExporter** [damienhaynes/moving-pictures#83]
```
The user should be able to resolve close matches in the GUI without going 
to the configuration panel. There are two ways this could be done. First it 
could be similar to TV-Series, a popup prompts you to make a selection, you 
do so and move on. This can be a bit intrusive though, so we should 
consider instead having a separate view mode that lists close matches, 
allowing the user to approve them or select the appropriate title. 
Basically a GUI version of the Movie Importer dialog, but streamlined a 
little bit.
```

Original issue reported on code.google.com by `conrad.john` on 2 Sep 2008 at 9:57
